ui/js/admin.js

```diff
@@ -1,42 +0,0 @@
-'use strict';
-
-/*exported admin*/
-var admin = angular.module('admin', [
-    'ui.router', 'ui.bootstrap', 'treeherder', 'react'
-]);
-
-admin.config(['$compileProvider', '$httpProvider', '$stateProvider', '$urlRouterProvider',
```

Comment:
There's another reference to `admin` in `lint.js` that can be removed too :-)

Reply:
Nice catch!  Thanks!